libs/SalesforceSDKCore/SalesforceSDKCore/Classes/IDP/SFSDKIDPAuthClient.m

```diff
@@ -106,6 +106,7 @@ - (BOOL)refreshCredentials {
 }
 
 - (BOOL)initiateLocalLoginInSPApp {
+    self.config.hideSettingsIcon = NO;
```

Comment:
Is that true? Should it always be shown in the SP app? Shouldn't we go with whatever the app wants?